src/Driver/API/SQLite/UserDefinedFunctions.php

```diff
@@ -16,10 +16,8 @@ final class UserDefinedFunctions
      *
      * @param int $a
      * @param int $b
-     *
-     * @return int
      */
-    public static function mod($a, $b)
+    public static function mod($a, $b): int
```

Comment:
In general, it might be risky: if the method doesn't enforce the parameter types, it may not always guarantee the return type. For instance:
```php
/**
 * @param int $value
 *
 * @return int
 */
function id($value) {
    return $value;
}
```
So each case has to be checked carefully

Reply:
I have checked all methods I've touched and only added the return type where I was certain.